packages/compass-query-bar/src/stores/query-changed-store.js

```diff
@@ -65,7 +65,7 @@ const configureStore = (options = {}) => {
         const copyable = state.lastExecutedQuery || this.getInitialState();
 
         for (const key in copyable) {
-          if (copyable.hasOwnProperty(key)) {
+          if (Object.prototype.hasOwnProperty.call(copyable, key)) {
```

Comment:
Fixes `error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins`